### ISSUE TITLE
problem in finetune stage

### ISSUE THREAD

**Issue by dianxin556** [MathiasGruber/PConv-Keras#12]

**Comment by dianxin556** [comment]
hi, i use your code to train my own model, but in funtune stage i set train_bn=False, the training loss began to increase rapidly and then decrese very slowly. and for a period end ,the predict image is black where there is the mask.Do you have the similar  situation when finetuning. Thanks a lot.

**Comment by MathiasGruber** [comment]
This happened in my training as well. I'm not sure why, but it's on my todo list to dig a bit more into the batch normalization layer of keras and figure out why this happens. If anyone already knows, please let me know :)

**Comment by EricZgw** [comment]
Have you solved this problem yet? @dianxin556 @MathiasGruber

**Comment by Squibit** [comment]
Did anyone figure this out yet? @dianxin556 @MathiasGruber @EricZgw 

Edit: And is it a problem? Will it eventually go away with enough finetuning?

**Comment by burhr2** [comment]
I fine-tuned based on phase1 only. My ROI is mostly at the centre of the image having a clean white background. train-800+ Val- 100+ and test-200+ with 0.95ssim for my test set
